radixdlt/babylon-wallet-android
# Delete account

This PR contains the delete account feature as a whole:

1. Delete button in account's settings
2. Special transaction review for account deletion
3. Deleting account (AKA tombstoning) when the transaction is successfull.
4. Syncing current accounts on profile with ledger to see if any of them is deleted. 
5. Deriving accounts will ignore not only hidden accounts but also deleted ones.

## How to test
1. Delete an empty account. Proceed to review and execute the account deletion. The account should be removed from wallet screen when successful.
2. Delete an account with assets. The same path is shown but now an account will also transfer assets to another account.
3. Do the same as with 2 but with an account with more than 50 resources. An error dialog will ask the user to transfer the assets manually.
4. Save the profile in a file and the proceed to delete an account. Then delete the wallet and import the profile. This should result in showing the "Previously Deleted Account Detected" and the profile should sync.
5. Try to recover a wallet with a seed phrase that used to associate with an account that had assets and is now deleted. This account should not appear in the derived accounts list.
6. Just create accounts (offline/online)
